app/src/pages/api/attachment/upload-url.ts

```diff
@@ -0,0 +1,29 @@
+import S3 from 'aws-sdk/clients/s3'
+import { NextApiRequest, NextApiResponse } from 'next'
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse
+) {
+  console.log("Using BUCKET_NAME:", process.env.BUCKET_NAME);
```

Comment:
Nit: please make sure your new files align with the code style convention in our package, 4 spaces for indentation. Spacing doesn't really matter, but within an existing codebase it's important to be consistent

Do you need to set up prettier in your IDE? It will autoformat so you don't need to think about this :)